### ISSUE TITLE
Simple downloads needed

### ISSUE THREAD

**Issue by alphapapa** [mozilla/Fira#160]
I wanted to download Fira on another computer.  So I googled Fira Mono and went to http://mozilla.github.io/Fira/.  There are examples of all of the fonts there, but not a single clickable link.

So I went to the links that were suggested last time I filed a bug about this: https://github.com/mozilla/Fira/releases.  The first link says "Fira Sans Condensed" in big letters.  I don't want to download the condensed version, so I scroll down and see "Fira Sans 4.106 & Mono 3.206."  Ok, those are the fonts I want, so I click that.

And I get an 80+ MB zip file containing every file in the entire project, including every font format, every technical report PDF, every source GLIF file, etc..  All I want are the TTF or OTF files for Fira Sans and Fira Mono!

Well, I thought I'd give the condensed version a try too, so I also downloaded that.  And I got a 100+ MB file also containing the entire project, because although the title said "Fira Sans Condensed," it was actually a new release of the entire project *including* the new condensed version.

Sure, I could download from fontsquirrel.com, or I could download from Google Web Fonts, but how do I know those sites have the latest releases of Fira?  I came here instead of going there because I want the fonts from the source, not through a middleman.

This is the official home of the Fira fonts, and it needs to have simple, official font downloads for users, not just huge archives of the entire project for font designers.  Every example on http://mozilla.github.io/Fira/ needs to have clickable download links to each font in OTF/TTF formats.

**Comment by easyaspi314** [comment]
@alphapapa Fira Sans Condensed isn't 100% complete; it doesn't have the kerning set and not all of the glyphs are done. 

If you want just the OTFs for mono, condensed, and sans (usually that will do for local use) you can see them at my dropbox.
https://www.dropbox.com/sh/4z2yjagjzx5swnd/AADZvc1RamX2NcWqqnw0rOfja?dl=0

**Comment by alphapapa** [comment]
Thank you, but I already downloaded the entire project twice, so I have all the files I need.  :)  This bug report is about providing simple downloads for new users.

**Comment by LMLB** [comment]
FYI: A zip file without the "technical reports" and "source" directories would be about 1/3 of the current size.

**Comment by mcswell** [comment]
FYI, there are zipped downloads here: https://carrois.com/typefaces/FiraSans/ (you have to click on the thingy in the upper right-hand corner that says "info and downloads").  BTW, the link to carrois.com on the front page of this github repo is broken, I guess it should be the one above.

**Comment by gluons** [comment]
Agree. I love **Fira** font. But can only download from release page that have size **100+MB**.
GitHub can upload binary file to release. So, they should upload final font files as **zip** in release menu.

~~Thanks [DownGit](https://minhaskamal.github.io/DownGit) for saving my time currently.~~
